### PR TITLE
DOCSP-37481 - Compatibility

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-37481
Staging - https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-37481-compat/compatibility/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/language-compatibility-table-pymongo.rst [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+.. [#three-six-compat] Pymongo 4.1 requires Python 3.6.2 or later.
+
+.. note::
+
+   PyPy3 is a Python 3.2-compatible alternative interpreter.
+
+Python 2
+~~~~~~~~
+
+.. list-table::
```
Comment (mongoKart) [inline source/includes/language-compatibility-table-pymongo.rst:396]: If you have any ideas on why this table's columns look weird, please let me know

Comment (ccho-mongodb) [reply source/includes/language-compatibility-table-pymongo.rst:396]: If referring to the column spacing, you can provide hints on the spacing by  using the  `widths` attribute ([Example](https://github.com/mongodb/docs-kafka-connector/blob/master/source/source-connector/configuration-properties/change-stream.txt#L35)).
Add the percentage for each column, e.g.

```
   :widths: 10 30 30 30
```

File: source/compatibility.txt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+MongoDB Compatibility
+---------------------
+
+The following compatibility table specifies the recommended version or versions
+of {+driver-short+} for use with a specific version of MongoDB.
+
+The first column lists the driver version.
+
+.. sharedinclude:: dbx/lifecycle-schedule-callout.rst
```
Comment (ccho-mongodb) [inline source/compatibility.txt:28]: Question:
Based on the MongoDB EOL lifecycle statement, is it necessary to show MongoDB versions prior to 3.4 in the compatibility tables?

**Review by ccho-mongodb — APPROVED** [review]
LGTM

File: source/includes/language-compatibility-table-pymongo.rst (hunk 1 above)
Comment (mongoKart) [reply source/includes/language-compatibility-table-pymongo.rst:396]: Thanks. That didn't fix it in the way I was hoping; I think the weirdness is a product of the table automatically placing the checkmark inline with the start of the column header. The only fix I can think of is manually putting the header text on two lines (`Python 2.5,\nJython 2.5`), but I couldn't make that work. Not a big deal.

Comment (ccho-mongodb) [reply source/includes/language-compatibility-table-pymongo.rst:396]: Oh I see. If the line break didn't render, I think you can use the "|" block operator, e.g.:

```
- | Python 2.5,
  |
  | Jython 2.5
```

**Review by ccho-mongodb — APPROVED** [review]
LGTM!

**Pull request merged**